plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftHiveMetastoreClient.java

```diff
@@ -283,6 +302,7 @@ public void setTableColumnStatistics(String databaseName, String tableName, List
                 statistics,
                 stats -> {
                     ColumnStatisticsDesc statisticsDescription = new ColumnStatisticsDesc(true, databaseName, tableName);
+                    catalogName.ifPresent(val -> statisticsDescription.setCatName(val));
```

Comment:
ditto

Reply:
@anusudarsan sorry did not understand.
Are you asking for this comment https://github.com/trinodb/trino/pull/21502#discussion_r1592148891 ?

Reply:
sorry. yes re: lamba usage